storybookjs/storybook
# Core: upgrade dotenv to 5.0.0

Issue: #2874 

Looks like major version bump was mostly a precaution: https://github.com/motdotla/dotenv/pull/237#issuecomment-341569785

## How to test
Open official-storybook. Make sure you don't see anything like this:
<img width="1378" alt="screen shot 2018-01-31 at 00 22 52" src="https://user-images.githubusercontent.com/6651625/35592148-30bf4be4-061d-11e8-9c51-8d21e6027a92.png">
